### PR TITLE
Add user list 

### PR DESCRIPTION
**Pull request by gqkc**
adding user list handler

### CODE REVIEW

File: icubam/backoffice/handlers/base.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import tornado.locale
```
Comment (fpquintao) [inline icubam/backoffice/handlers/base.py]: this file already exists in the codebase. Can you sync your client?

File: icubam/backoffice/handlers/home.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from tornado_sqlalchemy import SessionMixin
```
Comment (fpquintao) [inline icubam/backoffice/handlers/home.py]: same here; file already exists.

File: icubam/backoffice/server.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import os.path
```
Comment (fpquintao) [inline icubam/backoffice/server.py]: same here

**Comment by dulacarnold** [comment]
@fpquintao should we close or keep this open?

**Comment by fpquintao** [comment]
Can we keep it open for now?
There are some files here that will be useful soon.

**Comment by fpquintao** [comment]
I will close this branch since we have one "list users" in place in the backoffice right now.

**Pull request closed without merging**